### ISSUE TITLE
[4.0] Displaying a text when editing an article

### ISSUE THREAD

**Issue by Magnytu2** [joomla/joomla-cms#30429]
### Steps to reproduce the issue
The text in the second column is not on a single line.

### Expected result
Text must be on one line

### Actual result
![Articles__New_-_Joomla_4_pour_Safari_-_Administration](https://user-images.githubusercontent.com/26197929/90773407-1fb4db80-e2f6-11ea-98f6-c5ff1cef0a81.jpg)

**Comment by infograf768** [comment]
Issue comes from default width
```
.control-group .control-label {
    width: 240px;
    padding: .3rem 1rem .3rem 0;
}
```

**Comment by infograf768** [comment]
Here we could do
```
.control-group .control-label {
    width: auto;
    padding: .3rem 1rem .3rem 0;
    min-width: 240px;
}

```
we have to test of any impact.

Also for that peculiar tab, if changed as done above, we dont need 3 columns anymore.
`<div class="column-count-md-2 column-count-lg-2">`
would fit and we would get
<img width="953" alt="Screen Shot 2020-08-20 at 15 16 53" src="https://user-images.githubusercontent.com/869724/90774739-3bb97c80-e2f8-11ea-8e98-0f2e64f9823b.png">

so we have first to see if width auto and mini-width are fine all over admin

**Comment by Magnytu2** [comment]
Sur Firefox, nous sommes sur de colonnes. Sur Safari trois.
![Articles__New_-_Joomla_4_pour_Safari_-_Administration-2](https://user-images.githubusercontent.com/26197929/90775471-2ee95880-e2f9-11ea-881a-5acf13a3408f.jpg)

**Comment by Quy** [comment]
See https://github.com/joomla/joomla-cms/pull/28790#issuecomment-626170938 for a possible issue with 2 columns.

**Comment by infograf768** [comment]
not only in 2 columns, but everywhere we have an hr (which is not the case here.

<img width="1418" alt="Screen Shot 2020-08-20 at 15 34 35" src="https://user-images.githubusercontent.com/869724/90776754-ca2efd80-e2fa-11ea-8991-4af302c6b331.png">

**Comment by Quy** [comment]
It is an issue under Global Configuration > Banners > Client

![30429](https://user-images.githubusercontent.com/368084/90787336-7c45d580-e2b9-11ea-936e-d572d56fe4d8.png)

**Comment by infograf768** [comment]
ok then. No perfect solution I guess.

**Comment by Quy** [comment]
Fixed in #30474
